lib/test.js

```diff
@@ -134,7 +138,11 @@ Test.prototype.run = function () {
 	}.bind(this));
 };
 
-Test.prototype.end = function () {
+Test.prototype.end = function (err) {
+	if (arguments.length >= 1 && Boolean(err)) {
+		this.ifError(err);
```

Comment:
If they've used `t.plan()`, won't this increase their assertion count causing a `plan error` message?